lib/fdo.c

```diff
@@ -1542,9 +1552,18 @@ static bool _STATE_TO2(void)
 			g_fdo_data->current_rvto2addrentry = g_fdo_data->current_rvto2addrentry->next;
 
 		}
+		if (0 == port) {
```

Comment:
Why are we setting this value in multiple places?

Reply:
One is during To1 for RV port and To2 for Owner port.